ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -69,9 +77,18 @@ public boolean onFling(final MotionEvent e1, final MotionEvent e2, final float v
     private View mActiveCard;
     private int mActiveCardPosition = 0;
 
+    private ObjectAnimator mBounceAnimator;
```

Comment:
it probably would make the logic much easier if you just set the bounce animator on the already existing `mAnimation` field

Reply:
How would I be able to stop it when a different animation runs.

Reply:
This logic right here already cancels any currently running animation: https://github.com/CruGlobal/godtools-android/blob/b7cff6e64839faa7e59a4c8b41bc08d2073cf469/ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java#L316-L321

Reply:
This also means you wouldn't need to add the `onStartAnimation` callback.